src/components/Composer/index.js

```diff
@@ -251,20 +250,34 @@ class Composer extends React.Component {
         } catch (e) {}
     }
 
+    /**
+     * Manually place the pasted HTML into Composer
+     *
+     * @param {String} html - pasted HTML
+     */
+    handlePastedHTML(html) {
+        const parser = new ExpensiMark();
+
+        this.paste(parser.htmlToMarkdown(html));
+    }
+
     /**
      * Check the paste event for an attachment, parse the data and call onPasteFile from props with the selected file,
      * Otherwise, convert pasted HTML to Markdown and set it on the composer.
      *
      * @param {ClipboardEvent} event
      */
     handlePaste(event) {
+        if (event) {
```

Comment:
do we have any case where this will be false?

Reply:
I think we don't need that, I'll remove the condition.